### PR TITLE
feat: amplify-flutter isConfigured getter

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Adds documentation for amplify-flutter 'isConfigured' API.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Ashish-Nanda — CHANGES_REQUESTED** [review]
We need to update the code snippets as well for the getting started and category setups.

File: docs/lib/project-setup/fragments/flutter/create-application/40_verifyAmplifyLibraries.md [hunk 1]
```diff
@@ -46,3 +46,9 @@ class _MyHomePageState extends State<MyHomePage> {
 ```
 
 Note that all calls to `addPlugin()` or `addPlugins()` are made before `Amplify.configure()` is called.
+
+If you need to test whether Amplify has already been configured successfully, you can use the `isConfigured` getter:
+
+```dart
+ var isConfigured = Amplify.isConfigured;
```
Comment (drochetti) [inline docs/lib/project-setup/fragments/flutter/create-application/40_verifyAmplifyLibraries.md:53]: general question, I was playing with Amplify/Flutter and realized that in our docs we're not very consistent when declaring variables, like recommended here: https://dart.dev/guides/language/effective-dart/usage#do-follow-a-consistent-rule-for-var-and-final-on-local-variables

it seems like `final` with type inference should be used more often, but I see `var` and the explicit type declared often in our snippets. Do we follow a pattern?

Comment (haverchuck) [reply docs/lib/project-setup/fragments/flutter/create-application/40_verifyAmplifyLibraries.md:53]: @drochetti We probably do not follow a consistent pattern (but obviously we should).

**Comment by haverchuck** [comment]
@Ashish-Nanda I don't know that we need to clutter all of the category documentation with this check. Perhaps we can make that change if customers continue to have issues?

**Review by Ashish-Nanda — APPROVED** [review]

**Pull request closed without merging**